app/src/components/TracksyWrapper.test.tsx

```diff
@@ -130,6 +138,25 @@ describe('TracksyWrapper', () => {
             })
         })
 
+        describe('when demo is ready', () => {
```

Comment:
For me demo is not really ready as the URL is in the wrong format

Reply:
You're right, here the test uses a mock of `isDemoReady` so it's okay, but to avoid confusion I've changed it to a correct URL.